### PR TITLE
feat(worker): extract i18n instance from command serialization

### PR DESCRIPTION
**Pull request by djabarovgeorge**
### Screenshots

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

File: libs/application-generic/src/usecases/compile-email-template/compile-email-template.usecase.ts [hunk 1]
```diff
@@ -30,26 +26,12 @@ export class CompileEmailTemplate extends CompileTemplateBase {
 
   public async execute(
     command: CompileEmailTemplateCommand,
-    initiateTranslations?: (
-      environmentId: string,
-      organizationId,
-      locale: string
-    ) => Promise<void>
+    // we need i18nInstance outside the command on order to avoid command serialization on it.
+    i18nInstance?: any
```
Comment (djabarovgeorge) [inline libs/application-generic/src/usecases/compile-email-template/compile-email-template.usecase.ts:30]: I don't really like the duplicated comments. I thought about extracting to an object type and adding a comment there, but the flattened input feels more user-friendly.

To make it generic with flattened input parameters, one solution could be:

```typescript
export type CompileTemplateContainer<T_Command> = [
  command: T_Command,
  // We need i18nInstance outside the command to avoid command serialization in it.
  i18nInstance?: any
];

public async execute(...[command, i18nInstance]: CompileTemplateContainer<CompileTemplateCommand>)
```

However, this approach seems to add more complexity, and I'm not sure I want to implement it.

File: apps/worker/src/app/workflow/usecases/send-message/send-message-email.usecase.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
-import { Injectable, Logger } from '@nestjs/common';
+import { Injectable, Logger, NotFoundException } from '@nestjs/common';
 import { ModuleRef } from '@nestjs/core';
-import * as inlineCss from 'inline-css';
+import inlineCss from 'inline-css';
```
Comment (scopsy) [inline apps/worker/src/app/workflow/usecases/send-message/send-message-email.usecase.ts:3]: @rifont, is that related to your recent changes?

**Review by scopsy — APPROVED** [review]
